blitz_api/services.py

```diff
@@ -84,11 +84,11 @@ def check_if_translated_field(field_name, data_dict):
     return False
 
 
-def getMessageTranslate(field_name, data_dict, both_required=False):
+def getMessageTranslate(field_name, data_dict, only_one_required=False):
```

Comment:
In fact we just want a minimum of 1 field, so it's not `both_required` but it's not `only_one_required` too.

It's more `required`, in the generic way, we need one of the lang, whatever the one we choose, or multiple.